charts/camunda-platform/templates/tasklist/deployment.yaml

```diff
@@ -73,7 +73,7 @@ spec:
                   name: {{ include "camundaPlatform.identitySecretName" (dict "context" . "component" "tasklist") }}
                   key: tasklist-secret
               {{- end }}
-            - name: CAMUNDA_TASKLIST_IDENTITY_AUDIENCE
+            - name: CAMUNDA_IDENTITY_AUDIENCE
               value: "tasklist-api"
```

Comment:
Will you follow up @houssain-barouni to make values like this configurable? for example when using Entra ID this value will be different to `tasklist-api` (this also applies to the client ID field)

Reply:
Hi @Ben-Sheppard , can you explain more please?
Do you mean Tasklist can be a M2M OAuth client with a different clientId and audience?

Reply:
@Ben-Sheppard I am not really aware of this usecase, can you do it all together along with other applications in https://github.com/camunda/camunda-platform-helm/pull/1155 so that it is all merged together

Reply:
or update this pull request if you wish

Reply:
I can indeed, lets close this PR and I'll pick it up in #1155